src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -50,6 +52,21 @@ def format_historical_data(data: SnubaTSResult) -> list[TimeSeriesPoint]:
     return formatted_data
 
 
+# return the start and end indices
+def get_start_and_end(data: SnubaTSResult) -> tuple[int, int]:
+    start, end = -1, -1
+    indices_with_results = []
+    for i, datum in enumerate(data.data.get("data", [])):
+        if "count" in datum:
+            indices_with_results.append(i)
+    if not indices_with_results:
+        return start, end
+    else:
+        start = indices_with_results[0]
```

Comment:
Ahhh grab the first and the last because the data is already ordered is that right?
Maybe we should add some quick validations in here for that?

Reply:
Yeah, we're looping through the data in order, so start is guaranteed to be <= end

Reply:
I wouldn't be able to test the validation—should I still add it?

Reply:
maybe a quick assertion here so that it would raise an exception 🤷 

could just be a quick `assert start < end`
This way it will raise in case this is ever false